pkg/httputil/httputil.go

```diff
@@ -45,12 +45,12 @@ func NewClient(credential *security.Credential) (*Client, error) {
 	}, nil
 }
 
-// IsFiltered return true if the given feedState matches the whiteList.
-func IsFiltered(whiteList string, feedState model.FeedState) bool {
-	if whiteList == "all" {
+// IsListState return true if the given feedState matches the listState.
+func IsListState(listState string, feedState model.FeedState) bool {
```

Comment:
Can we bind this function to the FeedState instead of being a util metohd, it looks like it is a method that is strongly bound to the FeedState, so it shouldn't be a util metohd?

Reply:
done